schedule.go

```diff
@@ -168,7 +168,7 @@ func runSchedules(a *App) {
 			sched = popSchedule(a)
 		}
 
-		log.Println("Next schedule:", sched.nextRunTime)
+		slog.Info("Next schedule", "next run time", sched.nextRunTime)
```

Comment:
nit: swap "next run time" for "start_time"

Reply:
changed